types/toastr/index.d.ts

```diff
@@ -373,9 +350,7 @@ interface Toastr {
 	 *
 	 * @param callback The function which will be passed the event details.
 	 */
-	subscribe: (callback: (response: ToastrResponse) => any) => void;
-
-	[key: string]: any;
```

Comment:
Is there a particular reason why you removed this? I don't think you mentioned it in your PR description, but I might be missing something.

Reply:
Hi, thanks for taking the time to review this. 
Yes, I intentionally removed the any-indexer. These any-indexers suppress errors that would normally appear when you attempt to use properties and methods that don't exist. They're only really needed (as a crutch) if the interface is incomplete, and this interface is complete. See [the toastr.js](https://github.com/CodeSeven/toastr/blob/50092cc604850a16c985520b63df184d3e0b4086/toastr.js#L27-L37)

Reply:
Great, just wanted to check. Thanks!